tools/release/release.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  releaseChangelog,
+  releasePublish,
+  releaseVersion,
+} from 'nx/src/command-line/release';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import yargs from 'yargs';
+
+void (async () => {
+  try {
+    const options = await yargs
+      .version(false)
+      .option('version', {
+        description:
+          'Explicit version specifier to use, if overriding conventional commits',
+        type: 'string',
+      })
+      .option('dryRun', {
+        alias: 'd',
+        description:
+          'Whether or not to perform a dry-run of the release process, defaults to true',
```

Comment:
[Proofreading] Super nitpicky I know, but the _"or not"_ here and below is redundant.

```suggestion
          'Whether to perform a dry-run of the release process, defaults to true',
```